src/Core/FieldHash.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
```

Comment:
I introduced this but it's not used. We could remove it or keep it in case we want to use std::unordered_map / std::unordered_set of DB::Field. I did not include it into Field.h itself to avoid adding more dependencies to a file that's used in almost 3000 other files.

Reply:
Nevermind, the style did not like it, so I've removed it.